### DISCUSSION TITLE
Ability to set Date Range Picker options overrides

### DISCUSSION THREAD

**Discussion by darren-glanville** [malzariey/filament-daterangepicker-filter#33] [Ideas]
I would like to use Date Range Picker across my whole app.

I would like to be able to use the **singleDatePicker** option and to be able to add my own ranges.

Would it be possible to add this in please?

**Comment by malzariey** [comment]
The Date Range Picker functionality does not support the singleDatePicker option . It is designed to focus only on a date range selection in the current version.
